platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,9 +88,29 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
-### AWS S3 CORS configuration
+## Download multiple files
 
-Apply a [CORS configuration](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ManageCorsUsing.html) to enable file uploads from Seqera to specific S3 buckets. The CORS configuration is a JSON file that defines the origins, headers, and methods allowed for resource sharing requests to a bucket. Follow [these AWS instructions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enabling-cors-examples.html) to apply the CORS configuration below to each bucket you wish to enable file uploads for:
+You can download up to a maximum of 1,000 files using the browser interface, or an unlimited number of files with the auto-generated download script that uses your cloud provider's CLI and credentials.
```

Comment:
small note: this `1000` limit is configurable. Its currently set to that value in theory it would be possible to change it. Not sure if these kind of details are usually included in the docs

Reply:
We can certainly add a note indicating that this is a configurable limit but then we'd need to have some sort of maximum value. It's not ideal to say it's adjustable without knowing what the upper limit is, imo.

I propose we let support know about this in case customers want to change the value. @alvaromartmart, where is this value found so that it can be changed?

Reply:
This can be a separate PR/discussion btw. 

Tagging @starnsd for visibility into this thread.

Reply:
Created https://seqera.atlassian.net/browse/EDU-161 to track this and resolving this conversation so we can merge.